golang/server/server.go

```diff
@@ -172,3 +205,16 @@ func (server MinimalGRPCServer) loadTlsCredentials() credentials.TransportCreden
 	}
 	return credentials.NewTLS(tlsConfig)
 }
+
+func isGracefulExit(err error) bool {
+	switch err {
+	case nil, http.ErrServerClosed, cmux.ErrListenerClosed, cmux.ErrServerClosed:
+		// do nothing, normal exit
+		return true
+	default:
+		if strings.Contains(err.Error(), "use of closed network connection") {
```

Comment:
can you explain what this is?

Reply:
can this be a case of its own?

Reply:
don't understand the second comment; will keep this for now; as mentioned in the comment above that i linked.

Reply:
sorry i was halucinating about how the `case` statement works